app/models/pet.rb

```diff
@@ -23,4 +23,8 @@ def self.all_pets_by_adoption_status
     Pet.order(:adoption_status)
   end
 
+  def self.pets_with_approved_apps
+   Pet.joins(:pet_applications)
+      .where("pet_applications.approved = 'true'")
```

Comment:
Really glad we figured out the pet_applications approval, so things like this are possible. Looks good!